apexlog/apexlog.go

```diff
@@ -0,0 +1,143 @@
+package apexlog
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/airbrake/gobrake/v5"
+
+	"github.com/apex/log"
+)
+
+// severity identifies the sort of log: info, warning etc. It also implements
+// the flag.Value interface. The -stderrthreshold flag is of type severity and
```

Comment:
* How do we use this flag with this integration?
* Why does it need to implement the `flag.Value` interface?